### ISSUE TITLE
Add static json file for console services and return them by an API

### ISSUE THREAD

**Issue by harsh-jindal** [fossasia/susi_server#1368]
**Actual Behaviour**
Not present.

**Expected Behaviour**
List the different console services across clients.

**Would you like to work on the issue?**
Yes

**Comment by Orbiter** [comment]
I don't think we should make console services as defined by json setup in the config available to the outside. I don't see a use case. How should that info be used?

**Comment by mariobehling** [comment]
This is a step forward listing the info on the public page and make console services testable, e.g. Play a song on youtube is a console service not listed on CMS. We just added it to the slideshow, but there is no skill listed. We want people to see the capabilities of SUSI.AI and currently the most advanced skills - console services - are not listed on CMS.
